pmm-app/src/pmm-dbaas/components/DBCluster/DBCluster.tsx

```diff
@@ -80,9 +80,9 @@ export const DBCluster: FC<DBClusterProps> = ({ kubernetes }) => {
     () => (
       <AddClusterButton
         label={Messages.dbcluster.addAction}
-        disabled={settingsLoading || !settings?.publicAddress}
-        showWarning={!settingsLoading && !settings?.publicAddress}
-        warningMessage={buildWarningMessage(styles.settingsLink)}
+        // disabled={settingsLoading || !settings?.publicAddress}
```

Comment:
maybe it's better to delete only `publicAddress`?

Reply:
We don't need disabled anymore because we only need it for address check